multinet/uploaders/nested_json.py

```diff
@@ -98,38 +99,31 @@ def upload(workspace: str, graph: str) -> Any:
     leaf_nodetable_name = f"{graph}_leaf_nodes"
 
     # Set up the database targets.
-    if space.has_collection(edgetable_name):
-        edgetable = space.collection(edgetable_name)
+    if loaded_workspace.has_table(edgetable_name):
+        edgetable = loaded_workspace.table(edgetable_name)
     else:
-        edgetable = space.create_collection(edgetable_name, edge=True)
+        edgetable = loaded_workspace.create_table(edgetable_name, edge=True)
 
-    if space.has_collection(int_nodetable_name):
-        int_nodetable = space.collection(int_nodetable_name)
+    if loaded_workspace.has_table(int_nodetable_name):
+        int_nodetable = loaded_workspace.table(int_nodetable_name)
     else:
-        int_nodetable = space.create_collection(int_nodetable_name)
+        int_nodetable = loaded_workspace.create_table(int_nodetable_name, edge=False)
 
-    if space.has_collection(leaf_nodetable_name):
-        leaf_nodetable = space.collection(leaf_nodetable_name)
+    if loaded_workspace.has_table(leaf_nodetable_name):
+        leaf_nodetable = loaded_workspace.table(leaf_nodetable_name)
     else:
-        leaf_nodetable = space.create_collection(leaf_nodetable_name)
+        leaf_nodetable = loaded_workspace.create_table(leaf_nodetable_name, edge=False)
```

Comment:
Again, this seems like it will cause us headaches with tables not containing what we expect. We should consider rejecting the upload if the tables exist.

Reply:
Opened #458.